sdk/core/core-auth/review/core-auth.api.md

```diff
@@ -17,6 +17,7 @@ export interface AccessToken {
 // @public
 export interface GetTokenOptions {
     abortSignal?: AbortSignalLike;
+    spanOptions?: any;
```

Comment:
consider `object` over `any`?

Reply:
That would give type errors when we access `parent` from span options.
At the moment we haven't gone through the design of what we want span options to contain.
We know for sure it can have a `parent`, but apart from that we haven't dug deeper here.

Right now, I would keep `any` to be in sync with the type for `spanOptions` we have in `core-http`

Next month, we will be revisiting tracing altogether. We will get back to what `spanOptions` should support then.

Reply:
> That would give type errors when we access parent from span options.

You might be confusing with `Object`. `object` [won't throw such errors](http://www.typescriptlang.org/play/index.html#code/GYVwdgxgLglg9mABMOcAUcBci4CMBWAptAJSIDeAUJQL7UD09iYcUOA1pSuiQNxeo0YEABsRfAenAATQsBhhC0idzQBGFYIBEAC0Ji4WiZUYdJacjU3pyiAIZgAngAUATnAAO2LQ8eIAbnYiIIRaiFb8lEA).